conf/references/production_reg_conf.pl

```diff
@@ -38,6 +38,9 @@
 
 # ---------------------- CURRENT CORE DATABASES---------------------------------
 
+# Load RR server first for priority
+Bio::EnsEMBL::Registry->load_registry_from_url("mysql://ensro\@mysql-ens-sta-5:4684/$curr_release");
+Bio::EnsEMBL::Compara::Utils::Registry::remove_multi();
```

Comment:
Just to confirm: one species won't be in both the rapid release and main site at the same time, right? Because it is already "painful" to have to handle a couple of clashes in metazoa, so I cannot imagine a whole list of reference species.